INSTALL

```diff
@@ -166,6 +166,11 @@ environment variables are set properly:
     # makes HOME and PYTHONPATH available for python
     runjob --exp-env HOME --exp-env PYTHONPATH ... : python script.py
 
+Compiling NEST with the GCC (`-DCMAKE_TOOLCHAIN_FILE=Platform/BlueGeneQ_GCC`)
+might require you to use a GSL library compiled by the GCC, otherwise undefined
+symbols break your build. Changing the compiler should be done in the file
+`<src>/cmake/Platform/BlueGeneQ_GCC.cmake`.
```

Comment:
The [`..._GCC.cmake`](https://github.com/nest/nest-simulator/blob/master/cmake/Platform/BlueGeneQ_GCC.cmake#L25) file indeed defines the `CMAKE_C_COMPILER` and `CMAKE_CXX_COMPILER`, otherwise cmake will find the gcc for the front-end and one always has to state the compiler flags in the configure line. BTW: the same goes for the [`..._XLC.cmake`](https://github.com/nest/nest-simulator/blob/master/cmake/Platform/BlueGeneQ_XLC.cmake#L22) file.